### PR TITLE
fix: block only students

### LINKED ISSUE #144: Review permissions for the role Open edX
The Open edX role grants users access to delete charts and doesn't provide access to the instructor dashboard.

We need to make sure course staff have read-only access to the default dashboards and charts already implemented, and that they have only access for what they need

### PR DESCRIPTION
**Pull request by Ian2012**
Fixes #144 

This PR fixes an issue that was blocking instructors and students from accessing the superset instance.

This PR removes the permissions that don't belong to the Instructor role.

### Screenshots

image

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by Ian2012** [comment]
@bmtcril should I remove the instructor access to list the charts and to see saved queries in the home page?

They already don't have access to those assets

File: tutoraspects/templates/aspects/apps/superset/pythonpath/openedx_sso_security_manager.py [hunk 1]
```diff
@@ -153,6 +153,11 @@ def get_courses(self, username, permission="staff", next_url=None):
             for course_id in next_courses:
                 courses.append(course_id)
 
+        # If the user has staff access to any courses, they are an instructor
```
Comment (bmtcril) [inline tutoraspects/templates/aspects/apps/superset/pythonpath/openedx_sso_security_manager.py:156]: I think you can simplify this whole block to:

```
       # If the user has no staff access to any courses, they are a student
        if not courses and bool("{{SUPERSET_BLOCK_STUDENT_ACCESS}}"):
            raise Exception(f"User {username} is not an instructor")
        
        return courses
```

**Comment by bmtcril** [comment]
> @bmtcril should I remove the instructor access to list the charts and to see saved queries in the home page?
> 
> They already don't have access to those assets

Makes sense to me, thanks for fixing this up

**Comment by Ian2012** [comment]
@bmtcril I wasn't able to remove these permissions without generating a really annoying error message:

image

**Comment by bmtcril** [comment]
Ugh, ok, we'll leave them then.

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-aspects&entry.867055334=https://github.com/openedx/tutor-contrib-aspects/pull/199&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-07-25+22:24&entry.1917517419=2023-07-26+16:46&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**